### ISSUE TITLE
Crash using the same setting like XMR stak 

### ISSUE THREAD

**Issue by bucah** [MoneroOcean/xmrig-amd#2]
I just tried using xmrig using the same thread setting (intensity, strided index, etc) like XMR stak. 
And the result is not stable. Within 2 hours the hash rate has drop the become zero.

Any help? 

FYI, I usually use XMR stak from the beginning

**Comment by setuidroot** [comment]
Yes, xmr-stak settings are not directly compatible with xmrig-amd.  Please post specs about your system, cards and settings and we can help you with the settings.  I started out with xmr-stak, but I've always had a slight edge in hashrate with xmrig and I've made many changes to it and compiled it with many different compilers.  IMO xmrig > xmr-stak > claymore (and all the others.)

I hope you've figured xmrig out anyways... post here if you want I can help or please close this issue.
